frameworks/kafka/docs/upgrade.md

```diff
@@ -0,0 +1,92 @@
+---
+post_title: Upgrade Kafka from 0.10.2.1 to 0.11.0.0
+menu_order: 10
+feature_maturity: preview
+enterprise: 'no'
+---
+
+# Upgrade Kafka from 0.10.2.1 to 0.11.0.0
+
+This document explains upgrading the DC/OS Kafka service from 0.10.2.x to 0.11.0.0 version. Please follow the Kafka [upgrade](https://kafka.apache.org/documentation/#upgrade) documentation for more information. 
+
+The rolling upgrade procedure (from 0.10.2.x to 0.11.0.0) is explained below. This is the recommended way of upgrading to 0.11.0.0 in order to guarantee no downtime while upgrading Kafka.
+
+
+1. Upgrade the code to the new version:
+   * Upgrade Kafka software to 0.11.0.0.
+   * Restart the brokers one at a time.    
+   * Wait until the entire cluster is upgraded.
+
+2. Change inter.broker.protocol.version to 0.11.0.0:
```

Comment:
Are there documented prerequisites, such as clients must support this version to use new features of the API?  Can we link out to those here?